### PR TITLE
Added missing DocumentationFile to net45 csproj

### PR DESCRIPTION
**Pull request by Minx-SigEp**
The .Net 4.5 project was not generating an XML documentation file to be
included in the NuGet package. This was causing documentation to be
unavailable in Visual Studio intellisense

### CODE REVIEW

File: package.csharp.nuget.cmd [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-ECHO ********** Build projects in release mode
-"C:\Program Files (x86)\MSBuild\14.0\Bin\msbuild.exe" Avalara.AvaTax.Net20.sln /t:Build /p:Configuration="Release"
-"C:\Program Files (x86)\MSBuild\14.0\Bin\msbuild.exe" Avalara.AvaTax.Net45.sln /t:Build /p:Configuration="Release"
-"C:\Program Files (x86)\MSBuild\14.0\Bin\msbuild.exe" Avalara.AvaTax.Portable.sln /t:Build /p:Configuration="Release"
-
-ECHO ********** Packaging NUGET
-cd src
-..\..\..\nuget.exe pack Avalara.AvaTax.nuspec
-popd
-pause
```
Comment (ted-spence-avalara) [inline package.csharp.nuget.cmd]: Looks like two changes got merged together here.  Can you keep each PR focused on one change at a time?  That makes it much easier to review.

File: src/Avalara.AvaTax.net45.csproj [hunk 2]
```diff
@@ -21,6 +21,7 @@
     <DefineConstants>TRACE;DEBUG;PORTABLE;NET45</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DocumentationFile>bin\Debug\net45\Avalara.AvaTax.RestClient.net45.XML</DocumentationFile>
   </PropertyGroup>
```
Comment (ted-spence-avalara) [inline src/Avalara.AvaTax.net45.csproj]: This looks correct

**Comment by ted-spence-avalara** [comment]
Does this PR address the issue with XML documentation files? https://github.com/avadev/AvaTax-REST-V2-DotNet-SDK/pull/24

**Comment by Minx-SigEp** [comment]
@ted-spence-avalara, Yes this was intended to address the issue with the XML documentation file missing in .net 45, but it looks like you already fixed this with PR #24 and then some. I think this Pull Request Can be closed.

**Comment by ted-spence-avalara** [comment]
Thanks for the feedback!  Thank you very much for bringing this to our attention.

**Pull request closed without merging**